### ISSUE TITLE
Images without alternative text

### ISSUE THREAD

**Issue by steverep** [home-assistant/frontend#14178]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

All `<img>` elements in the UI (36 occurrences) are missing the `alt` attribute, which makes them inaccessible to screen readers.

### Describe the behavior you expected

Image tags should always have an `alt` attribute.  If the image qualifies as "purely decorative" under WCAG, then it should be hidden using `alt=""`.

### Steps to reproduce the issue

n/a

### What version of Home Assistant Core has the issue?

2022.10.x

### In which browser are you experiencing the issue with?

n/a

### Which operating system are you using to run this browser?

n/a

**Comment by steverep** [comment]
Reopening as there are still a few `img` tags left to conquer

**Comment by steverep** [comment]
Still an issue

**Comment by steverep** [comment]
Still an issue
